### ISSUE TITLE
Feeds for "who is hiring" and "seeking freelancer" threads?

### ISSUE THREAD

**Issue by jaredandrews** [edavis/hnrss#21]
Hi,

Great work on this project. I am curious if you have any plans to add a feed for these threads: [who is hiring](https://news.ycombinator.com/item?id=15824597) and [seeking freelancer](https://news.ycombinator.com/item?id=15824598)? I would personally find this very useful.

I was just looking over the code for this app and I'm pretty sure I could implement it myself. I just wanted to make sure I wouldn't be stepping on your toes if I did. Do you accept PRs?

Thanks!

**Comment by edavis** [comment]
Hi Jared — Have you tried the search feature? It might already have what you need:

- https://hnrss.org/newest?q=Ask%20HN:%20Who%20is%20hiring%3F&search_attrs=title
- https://hnrss.org/newest?q=Ask%20HN:%20Freelancer%3F&search_attrs=title

I also recently added https://hnrss.org/jobs which mirrors https://news.ycombinator.com/jobs, if you're interested in that sort of thing.

But to answer your last question — yeah, totally open to PRs. If it's a major change maybe just drop me a note first, but other than that I'm happy to look over any PR.

Have a good one!

- Eric

**Comment by jaredandrews** [comment]
@edavis Hey! Thanks for the fast reply!

I started writing a response to this and then I needed to dig into the code to make sure I knew what I was talking about and then I ended out writing some code :O 

I have made a Pull Request: https://github.com/edavis/hnrss/issues/21#issuecomment-348732185 that provides additional context and an implementation of what I am trying to do. Check it out when you have a chance. Thanks!
